src/features/About/about.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './about.css';
+import { FaAward } from 'react-icons/fa';
+import { AiOutlineFieldTime } from 'react-icons/ai';
+import { VscFolderLibrary } from 'react-icons/vsc';
+import selfImage from '../../assets/Image/selfImage.jpg';
+
+const About = () => (
+  <section id="about">
+    <h5>
+      Get To know
+    </h5>
+    <h2>About Me</h2>
+    <div className="container about__container">
+      <div className="about__me">
+        <div className="about__me-img">
+          <img src={selfImage} alt="Self" />
+        </div>
+
+      </div>
+      <div className="about__content">
+        <div className="about__cards">
+          <article className="about__card">
+            <FaAward className="about__card-icon" />
+            <h5>Experience</h5>
+            <small> 2 + years</small>
+          </article>
+          <article className="about__card">
+            <AiOutlineFieldTime className="about__card-icon" />
+            <h5>Coding</h5>
+            <small> 1200+ Hours </small>
+          </article>
+          <article className="about__card">
+            <VscFolderLibrary className="about__card-icon" />
+            <h5>Projects</h5>
+            <small> 20 + completed</small>
+          </article>
```

Comment:
![image](https://github.com/Krishnabot/Portfolio-React/assets/116451144/609f7653-7844-44b9-8e04-17fac6231bc6)

- `optional` I belive you don't want to have an odd structure on your page?, can you either make the third container full-width or make in details drop down? what you think

- `optional`  same with this one
![image](https://github.com/Krishnabot/Portfolio-React/assets/116451144/c2c8c958-f4f6-4790-9773-10fca53d794e)